test/framework/e2e/tls/scanner.go

```
@@ -0,0 +1,449 @@
+							},
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      "results",
+									MountPath: "/tmp",
+									ReadOnly:  true,
+								},
+							},
+						},
+					},
+					Volumes: []corev1.Volume{
+						{
```

golangci-lint (2.12.2)

[error] composites: k8s.io/api/core/v1.Volume struct literal uses unkeyed fields

(govet)